src/main/resources/ath-container/Dockerfile

```diff
@@ -53,3 +53,6 @@ RUN touch /home/ath-user/.Xauthority
 # have in ENTRYPOINT as the container is started as ath-user.
 USER root
 RUN chmod ug+s "$(which docker)"
+
+# Make sure you can start vnc as root also
+RUN (echo ath-user; echo ath-user; echo "n") | vncpasswd 
```

Comment:
Does it not need any of the other *X-thingies* to be configured?

Reply:
Apparently no, the `.vnc` folder is created by the call to `vncpasswd` and I have used this as an agent run as root in my private instance (a K8s cluster) without problems.

However, now that you mention it seems strange, maybe we also do not need the extra thingies in the normal case?

I will perform some extra tests just to be sure